x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -176,6 +176,12 @@ export const basicPush = {
 
 export const pushedCase: Case = {
   ...basicCase,
+  connector: {
+    id: '123',
+    name: 'My Connector',
+    type: ConnectorTypes.jira,
```

Comment:
previously value was `ConnectorTypes.none`, which does not make sense for a case with pushed data